tasks/broad/GermlineVariantDiscovery.wdl

```diff
@@ -92,7 +92,7 @@ task HaplotypeCaller_GATK4_VCF {
     Boolean make_bamout
     Int preemptible_tries
     Int hc_scatter
-    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.1.4.0"
+    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.2.0.0"
```

Comment:
Was this change intentional? Isn't KCCG still using an earlier version?

Reply:
Yeah, intentional. The thing is that sharding the variant calling from CRAMs is not working with 4.1.4.0, when an interval spans 2 chromosomes.

Example:

```
gatk --java-options "-Xms4000m -XX:GCTimeLimit=50 -XX:GCHeapFreeLimit=10"   HaplotypeCaller   -R /hg38/Homo_sapiens_assembly38.fasta  -I gs://playground-au/cromwell/NA12878-full.cram  -L 40scattered-chr15-chr16.interval_list   -O test.g.vcf.gz   -contamination 0   -G StandardAnnotation -G StandardHCAnnotation -G AS_StandardAnnotation   -GQB 10 -GQB 20 -GQB 30 -GQB 40 -GQB 50 -GQB 60 -GQB 70 -GQB 80 -GQB 90   -ERC GVCF
```

4.1.4.0 here fails `when 40scattered-chr15-chr16.interval_list` spans chr15 and chr16:

```
...
chr15   101900000       101981189       +       . intersection ACGTmer.177
chr16   10001   99999   +       . intersection ACGTmer.1
```

4.1.4.0 works however if the interval list only contains intervals from a single chromosome.

4.2.0.0 works nicely either way.